migration-to-PERCS-v2.0.md

```diff
@@ -0,0 +1,222 @@
+# Migrating from PER-CS v1.0 (PSR-12) to PER-CS v2.0 ###
+
+## Summary
+
+PER-CS is the next evolution of the PSR set of Coding Standards from the
+PHP-FIG (Framework Interoperability Group). It extends the Coding Standards
+laid out in PSR-12 to the newest functionality added to PHP such as the match
+keyword, enums, attributes and more.
+
+This document describes the changes and additions on a section by section
+basis between PER-CS v2.0 and PER-CS v1.0 which is a direct equivalent of 
+PSR12 with very minor changes.
+
+It is derived in part from [a GitHub generated changelog](https://github.com/php-fig/per-coding-style/compare/1.0.0...2.0.0)
+and focuses on the changes in a section-by-section basis as it's focus is to be more readable.
+
+The intent of this document is to provide a summary of these changes that can
+then be used to drive action-lists for toolset producers to support PER-CS v2.0.
+
+Any section not specified in this document does not have any PER-CS specific changes.
+
+## [Section 2.6](https://www.php-fig.org/per/coding-style/#26-trailing-commas)
+Numerous constructs now allow a sequence of values to have an optional trailing
+comma:
+ * If the final item is on the same line then there MUST NOT be a trailing comma
+ * If the final item is not on the same line then there MUST be a trailing comma
+
+## [Section 4.4](https://www.php-fig.org/per/coding-style/#44-methods-and-functions)
+
+If a function or method contains no statements or comments (such as an empty
+no-op implementation or when using constructor property promotion), then the
+body SHOULD be abbreviated as {} and placed on the same line as the previous
+symbol, separated by a space.
+So, for a method of a subclass that does nothing:
+
+    <?php
+    class SubClass extends BaseClass
+    {
+        protected function init() {}
+    }
+
+## [Section 4.6](https://www.php-fig.org/per/coding-style/#46-modifier-keywords)
+
+Modifier keywords are keywords that alter how PHP handles classes,
+properties and methods.
+
+These keywords MUST BE ordered as follows:
+
+[abstract|final] [public|protected|private] [static] [readonly]
+[type] name
+
+Furthermore all keywords must be on a single line and MUST be separated
+by a single space.
+
+## [Section 4.7](https://www.php-fig.org/per/coding-style/#47-method-and-function-calls)
+
+Named arguments and method chaining
+
+When making a method or function call, there MUST NOT be a space between
+the method or function name and the opening parenthesis, there MUST NOT
+be a space after the opening parenthesis, and there MUST NOT be a space
+before the closing parenthesis.
+
+In the argument list, there MUST NOT be a space before each comma,
+and there MUST be one space after each comma.
```

Comment:
Are these actually new in 2.0?  I thought those were both very old, like PSR-2 old.